rboot-sampleproject/rom0.ld

```diff
@@ -207,4 +207,4 @@ SECTIONS
 }
 
 /* get ROM code address */
-INCLUDE "../ld/eagle.rom.addr.v6.ld"
+INCLUDE "../rboot/eagle.rom.addr.v6.ld"

```

Comment:
In my case the linker failed on the file missing so I had to change this to point to an existing file...